300-quarkus-vertx-webClient/.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
Please remove all the `mvnw` related stuff from the PR.

we have it in the root of the project, no need to have this stuff duplicated in the sub-directories